frontend/src/ui-kit/components/CodeBlock/CodeBlock.tsx

```
@@ -0,0 +1,127 @@
+  code,
+  language,
+  showLineNumbers = false,
+  className = '',
+  title,
+}: CodeBlockProps) {
+  // Trim trailing whitespace from code
+  const trimmedCode = code.trimEnd();
+
+  return (
+    <div
+      role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)